src/shadowbox/model/access_key.ts

```diff
@@ -66,6 +66,8 @@ export interface AccessKeyRepository {
   removeAccessKey(id: AccessKeyId): boolean;
   // Lists all existing access keys
   listAccessKeys(): AccessKey[];
+  // Changes the default port for new access keys.
```

Comment:
-> "Changes the port for new access keys."

No need for "default"

Reply:
Done